### ISSUE TITLE
loss suddenly drop to 0 and remain 0

### ISSUE THREAD

**Issue by huxian0402** [dvlab-research/MiniGemini#42]
基于提供的1 stage预训练模型Mini-Gemini-7B-Pretrain和默认的配置scripts/llama/train/stage_2_full_v7b_672_hr_1536.sh，单独使用minigemini_generation_pure_text.json 训练2 stage，loss在训练过程中（10个step后）突然降至0，在后续训练过程中持续为0，

![image](https://github.com/dvlab-research/MiniGemini/assets/37959240/9dea2831-3ec2-4ce5-85a0-3f8752fbd4f4)

@yanwei-li @yukang2017 @wcy1122 打印出中间变量，看到shift_logits为Nan导致loss出现为Nan的情况，这个正常吗？

![image](https://github.com/dvlab-research/MiniGemini/assets/37959240/b771b59e-c4b6-4f1f-857d-d213fc130876)

**Comment by yanwei-li** [comment]
Hi, please ensure you are using the same environment of `transformers==4.39.0` and `deepspeed==0.11.1`. Different environments may cause this error.

**Comment by huxian0402** [comment]
@yanwei-li 感谢大老，对齐transformers==4.39.0 and deepspeed==0.11.1版本后，MLLM输出Nan的问题解决了
